### DISCUSSION TITLE
Installing genealogy in docker

### DISCUSSION THREAD

**Discussion by adamgas** [MGeurts/genealogy#23] [Ideas and feature requests]
I'd really like to see this app in containerized environment.

Right now, I have to learn how to install database (MySQL or MariaDb). Using containers it would be much easier.

Any ideas or plans on buliding a container, e.g. in Docker?

**Comment by chinmaypurav** [comment]
Check #30

**Comment by MGeurts** [comment]
Hi,

I'm not familiar with [ServerSideUp](https://serversideup.net/) nor with **Docker**.
So it's difficult for me to integrate stuff into the application I'm not familiar with myself.
I would not be able to maintain those integrations nor respond to questions asked about them.

I suggest you fork the application and modify it to your needs.
That way you can maintain the tools you added to your fork en respond to user request concerning Docker integration.

Kind regards,
Marc
